.github/workflows/build.yml

```diff
@@ -216,9 +216,12 @@ jobs:
       - name: Install oc
         id: install-oc
         if: ${{ steps.verify_requires.outputs.cluster_needed == 'true' }}
-        uses: redhat-actions/openshift-tools-installer@v1
-        with:
-          oc: latest
+        run: |
+          wget --progress=dot:mega https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz
```

Comment:
Probably don't need `--progress` in CI.

Reply:
Edit: Found the `--quiet` option !

---

Here is how it looks like if you don't specify a progress option: https://github.com/mgoerens/test-github-workflow/actions/runs/13547227001/job/37861759632

I've settled for `--progress=dot:mega`: https://github.com/mgoerens/test-github-workflow/actions/runs/13547210704/job/37861708301

But we can also go to `--progress=dot:giga`: https://github.com/mgoerens/test-github-workflow/actions/runs/13547548815/job/37862815910

There's no way to completely remove progress outputs AFAIK. Note that Github actions (or at least the `ubuntu-22.04` image this runs on) are using wget 1.21.2, while my Fedora has wget 2.2.0 which seems to have a different set of options.